test/e2e/testing-helpers.js

```diff
@@ -0,0 +1,45 @@
+import { ClientFunction } from 'testcafe';
+
+export async function assertTable (t, table, expectedData) {
+    const columns = ['name', 'power'];
+
+    await t.expect(table.find('td').count, expectedData.length * columns.length);
+
+    let currentTdText = null;
+    let expectedText  = null;
+
+    for (let i = 0; i < expectedData.length; i++) {

```

Comment:
why not to use `for ...of` https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Statements/for...of